### ISSUE TITLE
Reduced stellar/DSO visibility when moon is up

### ISSUE THREAD

**Issue by Atque** [Stellarium/stellarium#978]
This is a purely visual enhancement request.

The Milky Way and Zodiacal light are both currently reduced in brightness once the moon is up (especially visible when there's a full moon), just like in reality. However, DSO textures and stars are just as visible when there's full moon as they are when the moon is below the horizon, which is not the case in nature, where the stellar limiting magnitude and the ability to distinguish DSO's are severly affected by moonlight. In fact, a full moon contributes to as much "light pollution" as a Bortle 7 sky, and a half moon is roughly equal to a Bortle 4 sky. 

Please compare these screenshots:

Full moon is up:
![stellarium-021](https://user-images.githubusercontent.com/37028368/75034976-ad7e4e00-54ae-11ea-9494-7ef01f5c114f.png)

Moon is below horizon:
![stellarium-022](https://user-images.githubusercontent.com/37028368/75034983-b7a04c80-54ae-11ea-874c-c478531d3472.png)

Note the increased brightness of the Milky Way in the second image, but stars are not affected. Preferably, stars would also be less visible when the (full) moon is up.

**Comment by gzotti** [comment]
There is the dynamic adaptation for stars, but it seems to not do much when fov is large.

**Comment by Atque** [comment]
@gzotti Yes, but it is only used when the moon (or planet or whatever) is within frame. The moon really brightens up all of the sky, meaning it is also difficult to observe an object that is not in the immediate vicinity of the moon.

Also, it should be a little easier to observe stars and DSO when the moon is low in the sky, than when it is high. This is already present with the Milky Way and Zodiacal light.

**Comment by gzotti** [comment]
MW&ZL: I know, my code. Stars: predates me, but we may have another look into this.

**Comment by gzotti** [comment]
See further suggestions in #351.
